cucumber/_authenticators_common/features/support/authenticator_helpers.rb

```diff
@@ -11,9 +11,9 @@ module AuthenticatorHelpers
   )
 
   def validated_env_var(var)
-    ENV[var].tap do |env_var_value|
-      raise MissingEnvVariable, var if env_var_value.blank?
-    end
+    env_var_value = ENV[var]
```

Comment:
we want also want to add @ here so it is `@env_var_value`?

Reply:
not really. this data is not memoized and is calculated each time. `env_var_value` is merely a placeholder as we want to also log the data.